src/components/structures/MessagePanel.tsx

```diff
@@ -48,6 +48,8 @@ import Spinner from "../views/elements/Spinner";
 import TileErrorBoundary from '../views/messages/TileErrorBoundary';
 import { RoomPermalinkCreator } from "../../utils/permalinks/Permalinks";
 import EditorStateTransfer from "../../utils/EditorStateTransfer";
+import { logger } from 'matrix-js-sdk/src/logger';
```

Comment:
We usually group external imports at the top of the list.
That should probably end up being an ESLint rule. I'll create an issue for it